api/v1/clusterextensionrevision_types.go

```diff
@@ -190,17 +183,16 @@ type ClusterExtensionRevisionStatus struct {
 	// ClusterExtensionRevision.
 	//
 	// The Progressing condition represents whether the revision is actively rolling out:
-	//   - When status is True and reason is Progressing, the revision rollout is actively making progress and is in transition.
-	//   - When Progressing is not present, the revision is not currently in transition.
+	//   - When status is True and reason is RollingOut, the ClusterExtensionRevision rollout is actively making progress and is in transition.
+	//   - When status is True and reason is Retrying, the ClusterExtensionRevision has encountered an error that could be resolved on subsequent reconciliation attempts.
+	//   - When status is True and reason is Succeeded, the ClusterExtensionRevision has reached the desired state.
+	//   - When status is False and reason is Blocked, the ClusterExtensionRevision has encountered an error that requires manual intervention for recovery.
+	//   - When status is False and reason is Archived, the ClusterExtensionRevision is archived and not being actively reconciled.
 	//
 	// The Available condition represents whether the revision has been successfully rolled out and is available:
-	//   - When status is True and reason is Available, the revision has been successfully rolled out and all objects pass their readiness probes.
-	//   - When status is False and reason is Incomplete, the revision rollout has not yet completed but no specific failures have been detected.
+	//   - When status is True and reason is ProbesSucceeded, the revision has been successfully rolled out and all objects pass their readiness probes.
 	//   - When status is False and reason is ProbeFailure, one or more objects are failing their readiness probes during rollout.
-	//   - When status is False and reason is ReconcileFailure, the revision has encountered a general reconciliation failure.
-	//   - When status is False and reason is RevisionValidationFailure, the revision failed preflight validation checks.
-	//   - When status is False and reason is PhaseValidationError, a phase within the revision failed preflight validation checks.
-	//   - When status is False and reason is ObjectCollisions, objects in the revision collide with existing cluster objects that cannot be adopted.
+	//   - When status is Unknown and reason is Reconciling, the revision has encountered an error that prevented it from observing the probes.
```

Comment:
the docs here and the table reported in the PR description are not in sync, the table has more rows.